frontend/app/views/event/page.scala.html

```diff
@@ -107,7 +107,7 @@ <h1 class="event-masthead__name">@event.name.text</h1>
             case _: MasterclassEvent => {}
             case _ => {
                 <div class="event-content__image" itemprop="image" content="@event.socialImgUrl">
-                    <div class="delayed-image-load" data-src="@event.imgUrl" data-alt="Please upload an event image" data-class="responsive-img"></div>
```

Comment:
Hah. Just realised this means that all the alt images for the event detail images have been "Please upload an event image"